### DISCUSSION TITLE
OpenEvse don't detect plug since new firmware

### DISCUSSION THREAD

**Discussion by DylanVautrin** [OpenEVSE/open_evse#17] [Q&A]
Hello, i work on a project using OpenEvse, and i have an issue with the plug detection sinc i've flashed the newest firmware in the Atmega , i have two board, one in 8.2.2.T2 that don't detect when something is plugged, and another in 8.2.0.T2 that works fine. I've try approximately all the commit between 8.2.0.T2 and 8.2.2.T2 but the one's that don't detect still detect nothing when i plug something.
For now i don't want to update the other board because i'm affraid it's gonna do the same.
Do you have an idea of what going on? I can give more informations if needed.
Thanks

**Comment by jeremypoulter** [comment]
And if you revert back to the 8.2.0 firmware that works ok?

**Comment by DylanVautrin** [comment]
Hello, sorry for the delay, 
Yes, if i'm using an older version that i have dowloaded, it works , i tried many version between the 8.2.0 and the 8.2.2 on the stable branch but none work, only the one i've downloaded and modify a bit works.

**Comment by Dylan811806** [comment]
Hello, I am making some test again with the lastest firmware and it still don't work.. The EVSE stay in EVSE_STATE_UNKNOWN and so nothing append when i plug something on the T2 plug, someone have an idea please ? 
Thanks

**Comment by KipK** [comment]
Have you upgraded wifi module to latest stable too ? It's mandatory for 8.2.3 fw ( firmware stays in bootlock mode until it's unlocked by wifi module ).
